### ISSUE TITLE
[PDF Export] 500 Status Code while requesting PDF

### ISSUE THREAD

**Issue by ThomasKaar** [StanfordBDHG/ENGAGE-HF-Firebase#25]
### Description

While cloning and setting up my local repository environment, I faced some issues when calling the PDF export Firebase function locally. Hence, I am creating an issue according to the [Contributing Guidelines](https://github.com/StanfordBDHG/.github/blob/main/CONTRIBUTING.md).

### Reproduction

After cloning the repo and following the following commands:
```
$ npm install
$ cd functions; npm install
$ npm run serve
```

and accessing the local URL for the PDF export function (in my case http://127.0.0.1:5001/stanford-bdhg-engage-hf/us-central1/exportHealthSummary), I received a 500 error message:
```
$ wget -O test.pdf http://127.0.0.1:5001/stanford-bdhg-engage-hf/us-central1/exportHealthSummary
...
2024-07-14 19:15:35 ERROR 500: Internal Server Error.
```

with an error message in the Firebase CLI of:
`⚠  functions: TypeError: this.doc.autoTable is not a function`

### Expected behavior

Status code 200 and exporting a PDF with dummy data (test.pdf)

### Additional context

Disclaimer: As this is my first time setting up and contributing to this repository, there might be something off with my setup.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

**Comment by pauljohanneskraft** [comment]
Thank you for bringing this up. Will be fixed in #19

**Comment by ThomasKaar** [comment]
Thanks for the quick response! 

I have been looking through the repository a bit, and I think it is pretty interesting to work on. I was looking through open issues and stumbled across #16, which seemed like a good issue as a first contribution. However, I also think this will be addressed in #19. 
Since I am new to the repository and not familiar with what is currently worked on and what would be great for a first contribution, I wanted to ask whether there are any other not-started issues or even new ones coming up that would be suitable as a first contribution to the project.

I wasn't sure whether this should be discussed here on the issue or a separate discussion in the [general StanfordBDHG GitHub Discussions](https://github.com/orgs/StanfordBDHG/discussions). In case this conversation should be moved, I am also happy to create a GitHub discussion around that.

Thanks already in advance!

**Comment by pauljohanneskraft** [comment]
Yes, there are already some tests in #19, but we might need more in the future for other functions / functionalities, especially since we are able to mock Firestore for invitations, etc after merging #19. I will let you know today or tomorrow about great beginner issues. Thank you for your help already!
